source/renderer/zettlr-editor.js

```diff
@@ -406,7 +415,7 @@ class ZettlrEditor {
 
     // We need to update citations also on updates, as this is the moment when
     // new spans get added to the DOM which we might have to render.
-    this._cm.on('update', (cm) => {
+    this._cm.on('update', () => {
```

Comment:
… is this a general behaviour of your linter that it simply strips of all parameters if they aren't used? Because I'd like to revert that. The reason: They don't take up that much real estate and then you know what you can work with if you have to refactor any of the callback functions. It's much easier not to forget some thing this way.

Reply:
Yes, this was done automatically. May I suggest you create format guidelines you like and reformat the code accordingly (using ESLint or whatever). In this way, other contributors can safely reformat according to these styles (If I hit reformat now in VS code, there are a lot of changes).